app/views/crime_applications/index.html.erb

```diff
@@ -37,5 +37,6 @@
         <% end %>
       </tbody>
     </table>
+    <%= paginate @applications %>
```

Comment:
I’ve seen there are some duplication of partials? I suppose it is to cope with the 2 different controllers we have (for in progress and for completed applications)? I’m no expert on kaminari but I believe there might be a way to configure the controller that kaminari will use perhaps so we reuse partials and code?

I believe something like `paginate @applications, params: { controller: 'foobar' }`

Maybe give it a try?

Reply:
No need for this anymore, read my other comment 👍